ksql-common/pom.xml

```diff
@@ -70,6 +70,12 @@
             <artifactId>spotbugs-annotations</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.3.1</version>
```

Comment:
I've noticed we tend to collect dependency versions at the top of the parent pom, rather than specifying them in individual dependencies like this. Perhaps worth moving the dependency (or at least the dependency version) into the parent pom, though I'm also new to pom organization and would defer if others have opinions on this.

(Whoops, just missed the merge...)

Reply:
Oops! Thanks for catching this... I'll tack this change on with whatever next PR I merge